meta-resin-common/recipes-containers/docker/docker_git.bb

```diff
@@ -18,10 +18,10 @@ DESCRIPTION = "Linux container runtime \
  subtle and/or glaring issues. \
  "
 
-SRCREV = "20f81dde9bd97c86b2d0e33bbbf1388018611929"
-SRCBRANCH = "v1.10.3"
+SRCREV = "670a205440471d72528f123ab7b686179d04e8e2"
+SRCBRANCH = "fix-docker-exec"
```

Comment:
This has to change to a release branch. It looks like all our tests are working so we'll going to merge on the resin-os/docker side and use the resulting branch